src/main/webapp/i18n/de/featureOverview.json

```diff
@@ -43,7 +43,7 @@
                 },
                 "anonymousAssessment": {
                     "title": "Anonyme Bewertung",
-                    "shortDescription": "Anonymes Bewertungsverfahren durch Korrektur Warteschlangen um Vorurteile zu vermeiden.",
+                    "shortDescription": "Anonymes Bewertungsverfahren durch Korrektur Warteschlangen, um Vorurteile zu vermeiden.",
```

Comment:
Should _Korrektur Warteschlangen_ be written in one word? -> _Korrekturwarteschlangen_

Reply:
I think Korrektur-Warteschlangen is the right word, it's also consistent with Bewertungs-Dashboard below